lib/components/artist/related_artists/related_artist.js

```diff
@@ -49,7 +49,18 @@ const styles = StyleSheet.create({
 });
 
 export default Relay.createContainer(RelatedArtist, {
-  initialVariables: { width: 150, height: 100 },
+  initialVariables: { imageWidth: 150, imageHeight: 100 },
+
+  prepareVariables: prevVariables => {
+    const {width, height} = Dimensions.get('window');
+    let w = width;
```

Comment:
I guess `prepareVariables` doesn’t update. Maybe there’s another Relay hook method? Or otherwise see if the dimensions are available at the top-level of the file.

```js
const windowDimensions = Dimensions.get('window');

export default Relay.createContainer(RelatedArtist, {
  initialVariables: { imageWidth: windowDimensions.width, imageHeight: windowDimensions/1.5 }
```